### DISCUSSION TITLE
posts/2023/06/about-r/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#11] [General]
# posts/2023/06/about-r/

https://guozheng.rbind.io/posts/2023/06/about-r/

**Comment by XiangyunHuang** [comment]
能帮助到你，我也很受激励，谢谢！统计之都论坛有很多nice的坛友，大家多交流。

**Reply by residualsun1** [reply]
好呀！！和大家认识是我的幸运。
